src/features/game/expansion/components/resources/Pebble.tsx

```diff
@@ -0,0 +1,254 @@
+    if (mined) setShowPebbleTimeLeft(true);
+  };
+
+  const handleMouseLeavePebble = () => {
+    setShowPebbleTimeLeft(false);
+  };
+
+  const strike = () => {
+    const isPlaying = sparkGif.current?.getInfo("isPlaying");
+
+    if (!isPlaying) {
+      miningAudio.play();
+
+      sparkGif.current?.goToAndPlay(0);
+
+      setTouchCount((count) => count + 1);
+
+      // On second strike, mine
+      if (touchCount > 0 && touchCount === HITS - 1) {
+        mine();
+        miningFallAudio.play();
+        setTouchCount(0);
+      }
+    } else return;
+  };
+
+  const mine = async () => {
+    setTouchCount(0);
+
+    try {
+      gameService.send("pebble.struck", {
+        index: pebbleIndex,
+      });
+      setCollecting(true);
+      minedGif.current?.goToAndPlay(0);
+
+      displayPopover(
+        <div className="flex">
+          <img src={smallStone} className="w-5 h-5 mr-2" />
+          <span className="text-sm text-white text-shadow">{`+${pebble.stone.amount}`}</span>
+        </div>
+      );
+
+      setToast({
+        icon: smallStone,
+        content: `+${pebble.stone.amount}`,
+      });
+
+      await new Promise((res) => setTimeout(res, 2000));
+      setCollecting(false);
+    } catch (e: any) {
+      displayPopover(
+        <span className="text-xs text-white text-shadow">{e.message}</span>
+      );
+    }
+  };
+
+  const timeLeft = getTimeLeft(pebble.stone.minedAt, PEBBLE_RECOVERY_TIME);
+
+  return (
+    <div
+      className="relative z-10 w-full h-full"
+      onMouseEnter={handleMouseHoverPebble}
+      onMouseLeave={handleMouseLeavePebble}
+    >
+      {/* Unmined pebble which is strikeable */}
+      {!mined && (
+        <div
+          ref={containerRef}
+          className="group cursor-pointer w-full h-full"
+          onClick={strike}
+        >
+          <Spritesheet
+            className="group-hover:img-highlight pointer-events-none z-10"
+            style={{
+              position: "absolute",
+              left: "-28px",
+              top: "-22px",
+            }}
+            getInstance={(spritesheet) => {
+              sparkGif.current = spritesheet;
+            }}
+            image={sparkSheet}
+            widthFrame={96}
+            heightFrame={64}
+            fps={24}
+            steps={6}
+            direction={`forward`}
+            autoplay={false}
+            loop={true}
+            onLoopComplete={(spritesheet) => {
+              spritesheet.pause();
+            }}
+          />
+        </div>
+      )}
+
+      <Spritesheet
+        style={{
+          position: "absolute",
+          left: "2px",
+          top: "-28px",
+          opacity: collecting ? 1 : 0,
+          transition: "opacity 0.2s ease-in",
+          imageRendering: "pixelated",
+        }}
+        className="pointer-events-none z-20"
+        getInstance={(spritesheet) => {
+          minedGif.current = spritesheet;
+        }}
+        image={dropSheet}
```

Comment:
<img width="146" alt="Screen Shot 2022-07-08 at 9 30 50 am" src="https://user-images.githubusercontent.com/11745561/177888112-13c39ba2-08e3-43db-8f40-d89c72e5fc85.png">

The green border on the stone looks strange on top of the water. I would suggest changing the art to use a grey that has some transparency so it would look good on top of water or on top of land.

Also, once the sprite sheet plays it just seems to stay there. Not sure if this should be working yet.